src/main/java/com/google/sps/servlets/DeleteOrganizationServlet.java

```diff
@@ -14,14 +14,52 @@
 
 package com.google.sps.servlets;
 
+
+import com.google.appengine.api.datastore.Entity;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import com.google.sps.data.GivrUser;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import com.google.sps.data.GivrUser;
-import java.io.IOException;
 
+/** Servlet responsible for deleting tasks. */
 @WebServlet("/delete-organization")
 public class DeleteOrganizationServlet extends HttpServlet {
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I expected a check that the user is a moderator of the organization or a maintainer here?

Reply:
Right now the organizations page only appears to moderators or maintainers and this button will only be shown on the organizations page (same with the edit button). Should there be an additional check here too though?